dirty-script.py

```
@@ -0,0 +1,60 @@
+
+
+def get_most_frequent_words(text, top_n=5):
+    words = clean_text(text).split()
+
+    word_counts = {}
+    for word in words:
+        word_counts[word] = word_counts.get(word, 0) + 1
+
+    sorted_words = sorted(word_counts.items(), key=lambda x: x[1], reverse=True)
+
+    some_var = [x[0] for x in sorted_words if len(x[0]) > 3]
```

Ruff (0.11.9)

Local variable `some_var` is assigned to but never used

Remove assignment to unused variable `some_var`

(F841)